.travis.yml

```diff
@@ -1,28 +1,34 @@
-# See https://github.com/silverstripe/silverstripe-travis-support for setup details
-
 sudo: false
 
 language: php
 
+env:
+  global:
+    - COMPOSER_ROOT_VERSION="4.0.x-dev"
+
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL CORE_RELEASE=4.0
+      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.0
-      env: DB=MYSQL CORE_RELEASE=4.0
+      env: DB=PGSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL CORE_RELEASE=4
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+    - php: 7.2
+      env: DB=MYSQL PHPUNIT_TEST=1
 
 before_script:
-  - git clone git://github.com/silverstripe/silverstripe-travis-support.git ~/travis-support
-  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require "silverstripe/cms"
-  - cd ~/builds/ss
+  - phpenv rehash
+  - phpenv config-rm xdebug.ini
+  - composer validate
+  - composer require --no-update silverstripe/installer:4.0.x-dev
+  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql:2.0.x-dev; fi
+  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
```

Comment:
May need some more tweaks, Travis is saying our requirements could not be resolved to an installable set of packages.
https://travis-ci.org/plastyk/silverstripe-dashboard/jobs/347232337